src/shared/menu/entries.ts

```diff
@@ -53,6 +52,7 @@ import {
     makeMenuLinkEntry,
     MenuBlock,
 } from "./helpers";
+import { toggleCodeBlock } from "../../rich-text/commands/code-block";
```

Comment:
import from `../../rich-text/commands` instead alongside the other commands for rich text mode.